util/execdetails/execdetails.go

```diff
@@ -371,6 +371,29 @@ func NewConcurrencyInfo(name string, num int) *ConcurrencyInfo {
 	return &ConcurrencyInfo{name, num}
 }
 
+// cacheInfo is used to save the concurrency information of the executor operator
+type cacheInfo struct {
+	hitNum   int
```

Comment:
Why do we need record hitNum?

Reply:
Do you mean that keeping `hitRatio` and `hitTotal` here is enough?